plugins/modules/cloud/spotinst/spotinst_aws_elastigroup.py

```diff
@@ -1379,7 +1432,7 @@ def main():
         availability_zones=dict(type='list', required=True),
         block_device_mappings=dict(type='list'),
         chef=dict(type='dict'),
-        credentials_path=dict(type='path', default="~/.spotinst/credentials"),
```

Comment:
Why not keep the default, and avoid manual processing?

Reply:
eeer.... I am sure I had a compelling motive for that at the moment, but now it makes little sense to me as well. Reverting that.